extensions/amp-share-tracking/0.1/test/test-amp-share-tracking.js

```diff
@@ -55,6 +56,8 @@ describe('amp-share-tracking', () => {
     xhrMock.onFirstCall().returns(Promise.resolve(mockJsonResponse));
     return getAmpShareTracking('http://foo.bar').then(ampShareTracking => {
       expect(ampShareTracking.vendorHref_).to.equal('http://foo.bar');
```

Comment:
this test case is made specifically for incoming fragment. there's no need to check other things here.
also, you don't need to pass in `http://foo.bar`